packages/kbn-test/src/functional_tests/tasks.js

```diff
@@ -35,6 +35,25 @@ in another terminal session by running this command from this directory:
 
 `;
 
+/**
+ * test if a list of argv flags tell the kibana server to run in the development environment
+ * @param {string[]} argv
+ * @return {boolean}
+ */
+function includesEnvNameDevelopmentFlag(argv) {
+  return argv.some((arg, i) => {
+    // find single-arg format
+    if (arg === '--env.name=development') {
+      return true;
+    }
+
+    // find split arg format
+    if (arg === '--env.name' && argv[i + 1] === 'development') {
```

Comment:
assumes they're always in this order?

Reply:
It wouldn't make sense if you passed `development --env.name`, it needs to be passed as `--env.name development` to be valid

Reply:
Ah! :+1: